Barsamyan-D/str-strip-plugin-david/str.css

```diff
@@ -1,217 +1,214 @@
 .structure-strip {
-    display: flex;
-    flex-direction: column;
-    gap: 1.5rem;
-    padding: 1.5rem;
-    border: 1px solid #e0e0e0;
-    border-radius: 8px;
-    background: #f8f9fa;
-    box-shadow: 0 2px 4px rgba(0,0,0,0.05);
+  display: flex;
+  flex-direction: column;
```

Comment:
There are some indentation differences automatically made when saving the change in the file..:((